test/exercises_test.dart

```diff
@@ -30,7 +30,9 @@ Future runCmd(List<String> cmds) async {
 Future<String> getPackageName() async {
   final pubspec = new File("pubspec.yaml");
 
-  final packageName = loadYaml(await pubspec.readAsString())["name"];
```

Comment:
I made the comment at wrong line, I meant to refer to line 31 and instead of var name `sourceUrl` we call it `pubspecString` since that's what it is and string is what `loadYaml` expects as its first arg.